### PR TITLE
remove fastapi dep to solve #47

### PR DESCRIPTION
**Pull request by xiangchenjhu**
remove unnecessary dependent 
resolves https://github.com/ssec-jhu/bluephos/pull/47

### CODE REVIEW

File: requirements/prd.txt [hunk 1]
```diff
@@ -5,5 +5,4 @@ ray[default]==2.9.3
 rdkit_pypi==2022.9.5
 torch==2.2.0
 torch_geometric==2.5.2
-fastapi==0.110.0
 uvicorn[standard]==0.29.0
```
[the remark this answers is not in the document]
Comment (xiangchenjhu) [reply requirements/prd.txt:8]: FastAPI can be removed as it is not in use in Bluephos project.

[the remark this answers is not in the document]
Comment (jamienoss) [reply requirements/prd.txt:8]: Yep, the above inline comment was on line 8 in reference to ``uvicorn``, which is probably only needed for ``fastapi``. If this is true, it should get removed in this PR also, no?

[the remark this answers is not in the document]
Comment (xiangchenjhu) [reply requirements/prd.txt]: Yes, I've also removed Unicorn after checking, and I've updated the PR accordingly.
Thanks!

**Review by jamienoss — APPROVED** [review]

**Comment by jamienoss** [comment]
@xiangchenjhu I see that there's an extra three commits to update with main and resolve conflicts, FYI, these shouldn't have been necessary. A good practice for updating your branch with main is to do the following (with conflicts in requirements/prd.txt):
```terminal
git fetch origin
git rebase origin/main
# resolves conflicts in requirements/prd.txt
git add requirements/prd.txt
git rebase --continue
git push origin del-fastapi-dep
# above should fail
git push origin del-fastapi-dep -f
```
If not familiar with rebasing, I always advise creating a backup of your branch first using (assuming already on branch)
```terminal
git branch del-fastapi-dep.bkup
```

**Comment by xiangchenjhu** [comment]
Thank you. I just noticed this helpful suggestion. It should be handy for future use.

**Pull request merged**